cns/restserver/ipam.go

```diff
@@ -655,6 +655,7 @@ func (service *HTTPRestService) GetExistingIPConfig(podInfo cns.PodInfo) ([]cns.
 		}
 	}
 
+	logger.Printf("[GetExistingIPConfig] IPConfigExists [%t] for pod [%+v]", ipConfigExists, podInfo.Key())
```

Comment:
can we print the whole podInfo vs just key

Reply:
podinfo is an interface and not a struct

Reply:
I recently implemented `String()` on the PodInfos: https://github.com/Azure/azure-container-networking/pull/2020
You can log it by using `%s` instead of `%v`